### PR TITLE
Fix line-wrapping in `flutter create` error message.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by andrewkolos** [comment]
Converting to draft since this does not look like it's ready for review (failing tests, no description). Let me know if that's not the case.

**Comment by Hixie** [comment]
Woops, I'd added a typo just before submitting it. D'oh.

**Comment by goderbauer** [comment]
(triage) talked to Ian and he said this is ready to review now.

**Review by andrewkolos — APPROVED** [review]
LGTM!

Before:
```
"asdf--923423" is not a valid Dart package name.

The name should be all lowercase, with underscores to separate words, "just_like_this".Use only basic
Latin letters and Arabic digits: [a-z0-9_].Also, make sure the name is a valid Dart identifier—that it
doesn't start with digits and isn't a reserved word.
See https://dart.dev/tools/pub/pubspec#name for more information.
```

After:
```
"asdf--923423" is not a valid Dart package name.

The name should be all lowercase, with underscores to separate words, "like_this". Use only basic Latin
letters and Arabic digits: [a-z0-9_], and ensure the name is a valid Dart identifier (i.e. it does not
start with a digit and is not a reserved word).

See https://dart.dev/tools/pub/pubspec#name for more information.

File: packages/flutter_tools/lib/src/commands/create_base.dart [hunk 1]
```diff
@@ -823,16 +823,15 @@ String? potentialValidPackageName(String name){
...
-      'The name should be all lowercase, with underscores to separate words, "just_like_this".',
-      'Use only basic Latin letters and Arabic digits: [a-z0-9_].',
-      "Also, make sure the name is a valid Dart identifier—that it doesn't start with digits and isn't a reserved word.\n",
-      'See https://dart.dev/tools/pub/pubspec#name for more information.',
-      if (potentialValidName != null) '\nTry "$potentialValidName" instead.',
-    ].join();
+    return '"$projectName" is not a valid Dart package name.'
+           '${ potentialValidName != null ? ' Try "$potentialValidName" instead.' : '' }\n'
+           '\n'
+           'The name should be all lowercase, with underscores to separate words, "like_this". '
```
Comment (andrewkolos) [inline packages/flutter_tools/lib/src/commands/create_base.dart:829]: Turbo-nit: would this be more clear?:

```suggestion
           'The name should be all lowercase, with words separated by underscores, "like_this". '
```

Or perhaps even

```suggestion
           'The name should be lowercase words separated by underscores, "like_this". '
```

Comment (andrewkolos) [reply packages/flutter_tools/lib/src/commands/create_base.dart]: Applied the lattermost suggestion in commit [tweak message](https://github.com/flutter/flutter/pull/150325/commits/d5ae5638916ba37ec8416007f7586a27f7f3dd54)

**Review by bkonyi — APPROVED** [review]

**Pull request merged**